doc-content/enterprise-only/admin-and-config/managing-business-central-process-administration-proc.adoc

```diff
@@ -0,0 +1,14 @@
+[id='managing-business-central-process-administration-proc']
+= Customizing process administration
+
+You can customize the default pagination option in {CENTRAL} by editing the _Default items per page_ property on the *Process Administration* page.
+
+.Procedure
+. Log in to {CENTRAL}.
+. Go to *Admin* -> *Process Administration*.
+. From the *Properties* section, update the _Default items per page_ property and click *Save*.
++
+[NOTE]
+====
+The allowed values for displaying the default items per page are 5, 10, 15, 20, 50 or 100.
```

Comment:
This needs to be fixed in the help text as well but allowed values are: 10, 20, 50 and 100.

Reply:
Changed as suggested.